### ISSUE TITLE
Display container name as window title

### ISSUE THREAD

**Issue by david-w-webb** [mozilla/multi-account-containers#2621]
### Before submitting a bug report

- [X] I updated to the latest version of Multi-Account Container and tested if I can reproduce the issue
- [X] I searched for existing reports to see if it hasn't already been reported

### Step to reproduce

Open a tab in a multi-account container.
Look at the window title displayed in the tray.

### Actual behavior

Firefox displays the title of the open tab as the title of the window in the system tray.

### Expected behavior

I would like for Firefox to display the open container name.

### Additional informations

This is a feature request.  I could not determine how to submit a feature request when looking at the FAQ, so I'm submitting as an issue, probably incorrectly.  Apologies.

I frequently run multiple Firefox windows, each with multiple tabs in the same container while managing multiple AWS accounts.  It is difficult to wade through all of the Firefox windows to locate the container I need to access.  It would be easier if, instead of displaying the title of the open tab, I could have Firefox display the title of the open container for that tab.
![firefox_tabs](https://github.com/mozilla/multi-account-containers/assets/16152666/fc978736-212e-43c1-87c3-dce4de2e7777)

**Comment by dannycolin** [comment]
> Look at the window title displayed in the tray.

Do you mean the list of Firefox windows opened in the taskbar on Windows?

Screenshot of a Windows taskbar for reference.
![image](https://github.com/mozilla/multi-account-containers/assets/7339076/2c0e1700-dfca-4613-a040-7cda43f4b537)

**Comment by david-w-webb** [comment]
> Do you mean the list of Firefox windows opened in the taskbar on Windows?

Yes.  I have my system tray/taskbar set up to display each instance of each window separately.  I run multiple monitors and each monitor other than the main one displays only the windows displayed on that monitor in the taskbar.  

In older versions of Windows, there would be a title bar for the browser which would display the title of the open tab as the browser title.  This is what is displayed in the taskbar.   

I'd like for the Multi-Account Containers plugin to include an option to override the window title with the name of the multi-account container for the open tab.  This would make it easy for me to identify which container each window is displaying.

**Comment by dannycolin** [comment]
Thanks for the clarification. I tagged it as needing change in Firefox itself since the addon is limited in what it can modify (pretty much only the tab title which is a bit of a hack). 

Another question, if you hover over the window in the window list, does a tooltip appears? If yes, does it contain the name of the container?

**Comment by david-w-webb** [comment]
Yes, if I hover over a system tray / taskbar icon, a set of thumbnail images for that program pops up and if I hover over one of them, the title bar text for that window displays.

**Comment by jikuja** [comment]
Agreed. This is important also for other operating systems. Is there issue on Firefox-side to follow?

<img width="292" alt="image" src="https://github.com/user-attachments/assets/ecbd316a-c122-4397-af8e-a0246c17fdd5">
